tests/python/driver/tvmc/conftest.py

```diff
@@ -110,10 +112,33 @@ def onnx_resnet50():
 
 @pytest.fixture(scope="session")
 def tflite_compiled_module_as_tarfile(tmpdir_factory):
+
+    # Not all CI environments will have TFLite installed
+    # so we need to safely skip this fixture that will
+    # crash the tests that rely on it.
+    # As this is a pytest.fixture, we cannot take advantage
+    # of pytest.importorskip. Using the block below instead.
+    try:
+        import tflite
+    except ImportError:
+        print("Cannot import tflite, which is required by tflite_compiled_module_as_tarfile.")
+        return ""
+
     target_dir = tmpdir_factory.mktemp("data")
     graph, lib, params, _ = get_sample_compiled_module(target_dir)
```

Comment:
Just find one very important place here. Do we still use old api of tvm.build? In new module-based api, we will only generate one `lib`, the `graph` and `params` is serialized into `lib`. Of course, we will still have one interface to make users get `params` separately to meet the need of network only updates weight only. We should notice that old api will be removed in the next release, so we shouldn't use it in the new feature.

Reply:
This refers to `compile`, which is the mechanism we are using here to generate this test fixture. The underlying API here is `relay.build` (not `tvm.build`, as far as I could check).

Here is the relevant snippet:

https://github.com/apache/incubator-tvm/blob/fde8d432acd2d7964607c4f9216720298db04007/python/tvm/driver/tvmc/compiler.py#L192-L203

With regards to the `params` embedded into `lib`, I think this is something we could improve in a future PR? It is likely to have more implications in the tool, that I'd need to investigate.

Reply:
My meaning is here:
https://github.com/apache/incubator-tvm/blob/fde8d432acd2d7964607c4f9216720298db04007/python/tvm/driver/tvmc/compiler.py#L215

We shouldn't rely on these `getter`, because this is just for old api compatibility and will remove them (except get params) in the next release. We should support to run the `lib` directly. If not in this pr, we should implement it in the next pr.

Reply:
I agree with @FrozenGene that we should use the up-to-date APIs. @leandron please fix it in this PR or add a TODO to fix it in a follow-up PR.

Reply:
Got it. As it will be deprecated in the next release, and I find it a bit risky to push a last minute change to the compiler (before the release), I'm gonna add the TODO for now, in the interest of risk and time.

We should soon update this soon, in a new PR to adjust `compile` and `run`.

Reply:
Add one TODO is acceptable for me.